### PR TITLE
Some minor styles refactors to ensure adherence to actual designs

### LINKED ISSUE #686: UAT: Sign In Button Style Incorrect
The sign in button in the left drawer for Venia is incorrect. The styling should be corrected to ensure adherence to design spec. 

Issue #1 from: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues

Button Location: 
1) Left Drawer, bottom 
2) Left Drawer > Sign In> Sign in form 

Fix to button should include: 
"Medium" size
Dark gray
Fill style

- [ x] `venia-concept`
- [ ] `pwa-buildpack`
- [x ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [ x] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### LINKED ISSUE #687: UAT: Correction to Required Field Indicator
Required field indicator should be corrected in order to ensure adherence to current design. 

All fields should be corrected to apply correct styling. 

Issue #2: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues

- [ x] `venia-concept`
- [ ] `pwa-buildpack`
- [ x] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [x ] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### LINKED ISSUE #688: UAT: Correct "Create an Account" button
Update styling to current version of "Create an Account" Button contained within the left drawer to apply updated design specs. 

Issue #4: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues

- [x ] `venia-concept`
- [ ] `pwa-buildpack`
- [ x] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [x ] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### LINKED ISSUE #689: UAT: correct "Forgot Your Password Link" color
Correct the text and color of the "forgot your password" link to adhere to current design spec. 

Corrections:
Update color to #C0123F 
Change text to read "Forgot password?"

Issue #5: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues

## This issue is for the following packages:

- [ x] `venia-concept`
- [ ] `pwa-buildpack`
- [ x] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [ x] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### LINKED ISSUE #692: UAT: Shopping Cart "More Actions" Indicator Corrections
The "More Actions" (kebab) indicator within the Shopping Cart is currently not correct according to design. 

Issue # 8: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues 

Corrections: 
Color #212121 
top-aligned to the text

## This issue is for the following packages:

- [x ] `venia-concept`
- [ ] `pwa-buildpack`
- [ x] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [ x] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### LINKED ISSUE #693: UAT: Correction to Checkout Subtotal text
Text and formatting for "Subtotal" are currently incorrect within the right nav drawer during checkout (when items are contained within cart). 

Issue 9:https://wiki.corp.magento.com/display/FF/Venia+UX+Issues

Current Version: "Subtotal (# Items) ($Amount)"
Correct version "Cart Total: ($Amount) (# Items)"

## This issue is for the following packages:

- [ x] `venia-concept`
- [ ] `pwa-buildpack`
- [x ] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [ x] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### LINKED ISSUE #694: UAT: Styling for "Checkout" Button is Incorrect
The current styling for the "Checkout" button in the right hand nav drawer (when items are added to cart) is incorrectly not in adherence to design. 

Issue 10: https://wiki.corp.magento.com/display/FF/Venia+UX+Issues

Corrections to be applied: 
"Medium" size
Dark gray
Fill style 

## This issue is for the following packages:

- [x ] `venia-concept`
- [ ] `pwa-buildpack`
- [ x] `peregrine`
- [ ] `pwa-devdocs`
- [ ] `upward-js`
- [ ] `upward-spec`

## This issue is a:

- [ x] Bug
- [ ] Feature suggestion
- [ ] Documentation issue
- [ ] Other (Please Specify)

### PR DESCRIPTION
**Pull request by AlexeyKaryka**
This PR closes #689, closes #692, closes #687, closes #693, closes #688, closes #686, closes #694.

## This PR is a:

- [ ] New feature
- [ ] Enhancement/Optimization
- [ X ] Refactor
- [ ] Bugfix
- [ ] Test for existing code
- [ ] Documentation

## Summary

When this pull request is merged, it will...

### CODE REVIEW

File: packages/venia-concept/src/components/MiniCart/miniCart.js [hunk 1]
```diff
@@ -95,15 +95,15 @@ class MiniCart extends Component {
...
+                            value={cart.totals.subtotal}
+                        />
                     </span>
                 </dt>
                 <dd className={classes.subtotalValue}>
-                    <Price
-                        currencyCode={cartCurrencyCode}
-                        value={cart.totals.subtotal}
-                    />
+                    {` (${cart.details.items_qty} Items)`}
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/MiniCart/miniCart.js]: Could you, please, move the value of this expression to local variable and use this variable here.

File: packages/venia-concept/src/components/SignIn/signIn.css [hunk 2]
```diff
@@ -14,7 +14,7 @@
 .forgotPassword {
     display: block;
     text-decoration: underline;
-    color: rgb(var(--venia-text-spot));
+    color: #c0123f;
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/SignIn/signIn.css]: Move this color to constants.

File: packages/venia-concept/src/components/MiniCart/miniCart.js [hunk 3]
```diff
@@ -95,15 +95,15 @@ class MiniCart extends Component {
             <dl className={classes.totals}>
                 <dt className={classes.subtotalLabel}>
                     <span>
-                        Subtotal
-                        {` (${cart.details.items_qty} Items)`}
+                        Cart Total :{` `}
```
Comment (Starotitorov) [inline packages/venia-concept/src/components/MiniCart/miniCart.js]: Could you, please, use `&nbsp;`.

Comment (AlexeyKaryka) [reply packages/venia-concept/src/components/MiniCart/miniCart.js]: Good idea, thanks

File: packages/venia-concept/src/components/MiniCart/miniCart.js [hunk 4]
```diff
@@ -91,20 +91,22 @@ class MiniCart extends Component {
...
+                            value={totalPrice}
+                        />
                     </span>
                 </dt>
                 <dd className={classes.subtotalValue}>
-                    <Price
-                        currencyCode={cartCurrencyCode}
-                        value={cart.totals.subtotal}
-                    />
+                    &nbsp;(${itemsQuantity} Items)
```
Comment (sirugh) [inline packages/venia-concept/src/components/MiniCart/miniCart.js]: Needing pluralization and lowercase.

File: packages/venia-concept/src/components/Input/input.css [hunk 5]
```diff
@@ -4,7 +4,7 @@
 }
 
 .requiredSymbol {
-    background-color: rgb(var(--venia-text-spot));
+    background-color: black;
```
Comment (sirugh) [inline packages/venia-concept/src/components/Input/input.css]: This does fix the color, however there looks to be an alignment issue:

Current: 
[Image from Gyazo](https://gyazo.com/747ecf967cd136feebf28a34f76148ff)

Desired:
[Image from Gyazo](https://gyazo.com/a66e22ded081db6932dbdc67cf5f32ff)

Comment (sirugh) [reply packages/venia-concept/src/components/Input/input.css]: Fixed.

[Image from Gyazo](https://gyazo.com/0727ee2e0b183ac2aba906654b159d9d)

File: packages/venia-concept/src/components/MiniCart/miniCart.js [hunk 6]
```diff
@@ -91,20 +91,23 @@ class MiniCart extends Component {
...
+                            value={totalPrice}
+                        />
                     </span>
                 </dt>
                 <dd className={classes.subtotalValue}>
-                    <Price
-                        currencyCode={cartCurrencyCode}
-                        value={cart.totals.subtotal}
-                    />
+                    &nbsp;({itemsQuantity} {itemQuantityText})
```
Comment (jimbo) [inline packages/venia-concept/src/components/MiniCart/miniCart.js]: Generally, we shouldn't be adding spacing in the content. We should try to render these nodes such that they're perfectly adjacent and then space them with CSS.

**Pull request merged**